src/find_deletions.cpp

```diff
@@ -13,6 +14,7 @@ void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments
     parser.info.short_description = "Find deletions";
     parser.info.version = "0.0.1";
     parser.add_positional_option(args.junction_file_path, "Input junctions tab-separated format.");
+    parser.add_positional_option(args.output_file_path, "The path of the output file.");
```

Comment:
I think we should use argument flags here:
```suggestion
    parser.add_option(args.junction_file_path, 'i', "input", "Input junctions tab-separated format.", seqan3::option_spec::REQUIRED);
    parser.add_option(args.output_file_path, 'o', "output", "The path of the vcf output file.", seqan3::option_spec::REQUIRED);
```
@eldariont: Or do you think that this is overhead because this is a call of a part of iGenVar and we will connect these program parts?

Reply:
I added them :)